datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -333,6 +333,8 @@ pub fn date_part(args: &[ColumnarValue]) -> Result<ColumnarValue> {
     }
     let (date_part, array) = (&args[0], &args[1]);
 
+    println!("date_part --> {:?}", date_part.data_type());
```

Comment:
This seems like a left over from debugging

Reply:
just fixed, apologize for the carelessness

Reply:
No problems!